nautobot/core/management/commands/generate_performance_test_endpoints.py

```diff
@@ -6,6 +6,21 @@
 
 from nautobot.core.utils.lookup import get_model_for_view_name
 
+# List of view names that are excluded for various error responses
+EXCLUDED_VIEW_NAMES = [
+    # "You do not have permission to perform this action." even when logged in as superuser
+    "apps-api:apps-list",
+    "plugins-api:plugins-list",  # same as above
+    "graphql-api",  # "Method \\"GET\\" not allowed."
+    "graphql",  # "Must provide query string."
+    "dcim-api:device-napalm",  # "No platform is configured for this device."
+    "dcim-api:connected-device-list",  # "Request must include \\"peer_device\\" and \\"peer_interface\\" filters."
+    "login",
+    "logout",
+]
+
+GET_ENDPOINT_SUFFIXES = ("-detail", "_list", "_notes", "_changelog", "-list", "-notes")
```

Comment:
The mixed `_` and `-` are disorienting to me. Can we put these in some logical order?